include/xsimd/types/xsimd_avx_int16.hpp

```diff
@@ -277,6 +277,12 @@ namespace xsimd
                 XSIMD_RETURN_MERGED_SSE(res_low, res_high);
 #endif
             }
+
+            template<bool... Values>
+            static batch_type select(const batch_bool_constant<value_type, Values...>& cond, const batch_type& a, const batch_type& b)
+            {
+              return select(cond(), a, b);
+            }
```

Comment:
Same remark as for avx512, maybe these methods can be moved to `avx_int_kernel_base`.